pkgs/os-specific/linux/hid-fanatecff/default.nix

```
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenv,
+  kernel,
+  fetchFromGitHub,
+  linuxConsoleTools,
+  bash,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hid-fanatecff";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "gotzl";
+    repo = "hid-fanatecff";
+    rev = lib.versions.majorMinor version;
+    hash = "sha256-1Nm/34Er/qfel9LJp++IWd7cTh2Wi93Kgd28YLMVvWo=";
+  };
+
+  nativeBuildInputs = kernel.moduleBuildDependencies;
+
+  makeFlags = kernel.makeFlags ++ [
+    "KVERSION=${kernel.modDirVersion}"
+    "KERNEL_SRC=${kernel.dev}/lib/modules/${kernel.modDirVersion}/build"
+  ];
+
+  patchPhase = ''
+    runHook prePatch
+
+    substituteInPlace fanatec.rules \
+      --replace-fail "/bin/sh" "${bash}/bin/sh"
+    substituteInPlace fanatec.rules \
+      --replace-fail "/usr/bin/evdev-joystick" "${linuxConsoleTools}/bin/evdev-joystick"
+
+    runHook postPatch
+  '';
```

```suggestion
  postPatch = ''
    substituteInPlace fanatec.rules \
      --replace-fail "/bin/sh" "${bash}/bin/sh"
    substituteInPlace fanatec.rules \
      --replace-fail "/usr/bin/evdev-joystick" "${linuxConsoleTools}/bin/evdev-joystick"
  '';
```
we don't want to break patches